src/tracker/psmove_tracker.c

```diff
@@ -414,7 +436,10 @@ PSMoveTracker *psmove_tracker_new() {
         camera = 0;
     }
 #endif
-
+#if defined(_WIN32) && defined(CAMERA_CONTROL_USE_PS3EYE_DRIVER)
+    printf("\nDetected PS3EYEDriver in Windows. You may have to `SET %s=2` (or =1).\n",
```

Comment:
It's a little more complicated than that. In Windows, when using libusb with PS3EYEDriver, two devices appear with the same VID and PID as the PSEye. As far as I can tell, there is no way to distinguish them without opening them. If you try to open the wrong one then the application crashes. I have been unable to get this to work except by using the environment variable and a little trial and error. I can change the wording from "You may have to" to "Please".

As discussed in the forum, the better solution would be to detect the camera at runtime and get rid of many of the preprocessor directives. Though that is a bigger job and also not a priority for me right now.